src/Nethermind/Nethermind.Dsl/Grammar/DslGrammar.g4

```diff
@@ -1,17 +1,19 @@
 grammar DslGrammar;
 
 tree: (expression)* ;
-expression : watchExpression | whereExpression | publishExpression | andCondition | orCondition ;
+expression : sourceExpression | watchExpression | whereExpression | publishExpression | andCondition | orCondition ;
+sourceExpression : SOURCE WORD ;
```

Comment:
why are you introducing sourceExpression?

Reply:
i'm actually not sure where that came from. May be a typo. I'll remove it.